asab/api/doc.py

```diff
@@ -38,55 +41,88 @@ def build_swagger_specs(self):
 				try:
 					adddict = yaml.load(docstr[i:], Loader=yaml.SafeLoader)
 				except yaml.YAMLError as e:
-						L.error("Failed to parse '{}' doc string {}".format(self.App.__class__.__name__, e))
+					L.error("Failed to parse '{}' doc string {}".format(self.App.__class__.__name__, e))
 			else:
 				description = docstr
 		else:
 			description = ""
 
-
 		specs = {
 			"openapi": "3.0.1",
 			"info": {
 				"title": "{}".format(self.App.__class__.__name__),
 				"description": description,
 				"contact": {
 					"name": "ASAB microservice",
-					"url": "http://www.github.com/teskalabs/asab",
+					"url": "https://www.github.com/teskalabs/asab",
 				},
 				"version": "1.0.0"
 			},
-			"servers": [
-				{"url": "../../"}  # Base path relative to openapi endpoint
-			],
-			"paths": {
+			"servers": [{"url": "../../"}],  # Base path relative to openapi endpoint
+			"paths": {},
+
+			# Authorization
+			"components": {
+				"securitySchemes": {
+					"oAuthSample": {
+						"type": "oauth2",
+						"description": "This API uses OAuth 2, this window allows [links](https://teskalabs.com)",
+						"flows": {
+							"authorizationCode": {
+								"authorizationUrl": "https://auth.test.loc/seacat/api/openidconnect/authorize",
+								"tokenUrl": "https://auth.test.loc/seacat/api/openidconnect/token",
```

Comment:
These URLs are hardcoded with custom values, that is incorrect.

- [ ] Use the configuration, put this here only if explicitly configured
- [ ] Document this in asab documentation

Reply:
Added and documented.